stories/s1grdiw.eocloud.stories.js

```diff
@@ -359,6 +359,23 @@ export const findDatesEPSG3857 = () => {
   return wrapperEl;
 };
 
+export const supportsProcessingAPI = () => {
+  const wrapperEl = document.createElement('div');
+  wrapperEl.innerHTML = "<h2>Supports Processing API</h2>";
+
+  const containerEl = document.createElement('pre');
+  wrapperEl.insertAdjacentElement("beforeend", containerEl);
+
+  const perform = async () => {
+    const layer = (await LayersFactory.makeLayers(`${DATASET_EOCLOUD_S1GRD.shServiceHostname}v1/wms/${instanceId}`, (lId, datasetId) => (layerId === lId)))[0];
+    const supportsProcessingAPI = layer.supportsApiType(ApiType.PROCESSING);
+    containerEl.innerHTML = JSON.stringify(supportsProcessingAPI, null, true);
```

Comment:
Weird... It shows `false` for me:

![image](https://user-images.githubusercontent.com/40981131/76401606-7222c100-6382-11ea-9a1d-de9b553b8933.png)

Did you run build?

> ...second argument should be a "A String or Number object", unless I'm missing something...

I think this is correct? Docs state for second parameter (`replacer`) this: "If this value is null or not provided, all properties of the object are included in the resulting JSON string." Am I missing something?

Reply:
Oh, this may also be because:

```
s1grdiw.eocloud.stories.js:399 
Uncaught (in promise) TypeError: Cannot read property 'supportsApiType' of undefined
```

Either wrong id of S1 GRD layer on EOCloud or getCapabilities on WMS  is still acting up

Reply:
Yes, I have. I get the same error as Žiga,.

Reply:
With `EOC_S1GRDIW_LAYER_ID=1_VV_ORTHORECTIFIED` it now works. Resolving.